crates/interledger-api/src/routes/accounts.rs

```diff
@@ -447,8 +451,20 @@ where
         .or(post_payments)
 }
 
+async fn consume_msg_drain(mut ws_rx: futures::stream::SplitStream<warp::ws::WebSocket>){
+    while let Some(result) = ws_rx.next().await {
+        let _ = match result {
```

Comment:
```suggestion
        if let Err(e) = result {
```
and the related small refactoring below

Reply:
Fixed.